src/OpenTelemetry/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -56,10 +56,12 @@ OpenTelemetry.Metrics.Metric.Name.get -> string
 OpenTelemetry.Metrics.Metric.Temporality.get -> OpenTelemetry.Metrics.AggregationTemporality
 OpenTelemetry.Metrics.Metric.Unit.get -> string
 OpenTelemetry.Metrics.MetricPoint
-OpenTelemetry.Metrics.MetricPoint.BucketCounts.get -> long[]
 OpenTelemetry.Metrics.MetricPoint.DoubleValue.get -> double
 OpenTelemetry.Metrics.MetricPoint.EndTime.get -> System.DateTimeOffset
-OpenTelemetry.Metrics.MetricPoint.ExplicitBounds.get -> double[]
+OpenTelemetry.Metrics.MetricPoint.GetBucketCounts() -> long[]
```

Comment:
as a follow up we need to do something similar to whats done for Tags. (readonly something)